packages/storybook/stories/dropdown-list.stories.tsx

```diff
@@ -115,21 +112,3 @@ export const WithDisabledOptions: Story = () => {
 
     return <DropdownList label="Select an option" options={disabledOptions} />;
 };
-
-export const ControlledWithSkipSelected: Story = () => {
```

Comment:
Par contre on n'a plus d'exemple où la valeur par défaut c'est de n'avoir aucune valeur.

Reply:
Je suis pas certain si je comprends bien ce que tu veux dire. Est-ce qu'on veut permettre que la valeur du component soit `undefined` ou on force d'avoir une valeur qui est présente dans la liste des options?

Avec un `<select>` natif, c'est possible de forcer `select.value = undefined`. Ça vide le texte du dropdown et `select.value` retourne alors `''`. Par contre, quand on réouvre la liste, la première option est déjà sélectionnée.

Perso, je voterais pour contraindre à la liste d'options. Par contre, ça reste possible de fournir une option `disabled` avec une valeur vide qui peut être sélectionnée par défaut.

Reply:
Ma réflexion derrière la question était motivée par le UI du comparateur financier. Je sais pas si on a ce genre de trucs ailleurs. En gros il y a un bouton pour dire qu'on préfère ne pas répondre, et ça ajoute et sélectionne une value vide.
![image](https://github.com/kronostechnologies/design-elements/assets/1082552/4f5c1258-06bd-4faa-9c6d-6cd84fd1e8ba)
https://moncomparateurfinancier.com/general-information

Reply:
Est-ce que c'était ça le use-case du _infamous_ `skipOption`? 😅 

Ça reste techniquement possible d'ajouter une valeur on-the-fly à la liste des `options`. On peut ensuite forcer la sélection de la nouvelle option par la prop `value`.

Par contre, je sais pas trop si c'est un comportement souhaité. Idéalement, on aurait d'avance une option `disabled` qui serait sélectionnée par défaut. Selon notre décision plus haut, on pourrait aussi forcer sa sélection avec `value`.

Reply:
Haha oui, c'était le use-case! J'ai l'impression que c'est la seule place que ça existait. On peut sûrement ignorer ce use-case là pour l'instant et on reverra les maquettes de MFB si jamais. J'ai pas envie de retarder et compliquer les choses pour un truc pas standard.

Reply:
Mais anyway, y'a un workaround en ajoutant une valeur dans l'array des options.